frontend/src/App.tsx

```diff
@@ -11,10 +13,13 @@ const theme = createMuiTheme({
 
 const App: React.FC = () => {
   return (
-    <ThemeProvider theme={theme}>
-      <CssBaseline />
-      <NavigationBar />
-    </ThemeProvider>
+    <Router>
+      <ThemeProvider theme={theme}>
+        <CssBaseline />
+        <NavigationBar />
+        <Routes />
```

Comment:
You should probably have a `<Layout>` component with children like we discussed.